cmd/containerd/command/service_windows.go

```diff
@@ -337,6 +339,9 @@ func initPanicFile(path string) error {
 	// and replace it.
 	if st.Size() > 0 {
 		panicFile.Close()
+
+		chunkedFileLog(path)
```

Comment:
Maybe I wasn't paying attention as well as I should have in standup 😆 but I thought for our fork we were just going to log the panic in full in one message. Wasn't the concern for upstream that we can't control how they have the shim configured so more than just stderr might end up in the file, but we know how ours is configured?

Reply:
there seems to be a limit on the message size. not sure if it's configured differently in prod, but locally I couldn't do 32KB, I'd get "Unable to parse" error.

Reply:
That might be the etw limit, but have we seen panic logs this large or you just fed it some arbitrarily large blob to test the max size you could send?

Reply:
(I think chunking it is fine btw, I was just trying to page in what we talked about)

Reply:
yeah, I've seen one at 300KB.

Reply:
Ok then we'll have to chunk it seems.